tests/lib/rules/implicit-arrow-linebreak.js

```diff
@@ -201,24 +201,24 @@ ruleTester.run("implicit-arrow-linebreak", rule, {
             errors: [UNEXPECTED_LINEBREAK]
         }, {
             code: `
-                (foo) =>
-                  // test comment
-                  bar
```

Comment:
Could these tests use something like the [`unIndent`](https://github.com/eslint/eslint/blob/af9688b7c4f6a3afe1b0ca5ba2f475c545e0309b/tests/lib/rules/indent.js#L57-L69) function that the `indent` tests use, rather than being aligned all the way to the left?

Reply:
@not-an-aardvark Just used the linked `unIndent` function and all tests are passing. I see that the `tests` folder doesn't have a `testhelpers.js` file. Would it be alright to move the above linked `unIndent` function into such a file? Otherwise, I will copy paste this into the tests for this rule.

Reply:
The upcoming commit does not do this, but can do so if agreed upon.

Reply:
I'd be fine with moving it to a separate file, but I think copy-pasting it is probably fine for now too.